File: 251004/Kalitsko/4lab/discussion/src/main/java/com/example/rest/repository/PostRepository.java
package com.example.rest.repository;

import com.example.rest.entity.Post;
import org.springframework.data.cassandra.repository.CassandraRepository;
import org.springframework.stereotype.Repository;

import java.util.Optional;

@Repository
public interface PostRepository extends CassandraRepository<Post, String> {
}

File: 251003/Pankratiev/rest/discussion/src/main/kotlin/bsuir/dc/discussion/repository/PostRepository.kt
package bsuir.dc.discussion.repository

import bsuir.dc.discussion.entity.Post
import bsuir.dc.discussion.entity.PostKey
import org.springframework.data.cassandra.repository.CassandraRepository
import org.springframework.stereotype.Repository

@Repository
interface PostRepository : CassandraRepository<Post, PostKey> {
}
